test/phabricator_tests/mock_phabricator.py

```diff
@@ -0,0 +1,237 @@
+try:
+    # Python 3 >
+    from unittest.mock import patch  # noqa: F401
+except ImportError:
+    from mock import patch  # noqa: F401
+try:
+    # Python 3.3 >
+    from unittest.mock import MagicMock  # noqa: F401
+except ImportError:
+    from mock import MagicMock  # noqa: F401
+
+"""
+This file mocks a lot of the Phabricator API calls
+"""
```

Comment:
The doctype of the whole module should be before imports. Here is an valid example of beginning the file

```python
#!/usr/bin/env python3
"""
Description on one line

More detailed description
"""
import json
import os
```